UPGRADE.md

```diff
@@ -262,6 +262,19 @@ return [
 Of course the `resourceKey` can still often be passed from a parent route and therefore omitted in the form route
 definition, which is often the case when making use of the `sulu_admin.resource_tabs` view.
 
+The frontend routes for a datagrid defined in the `Admin` classes now need the `datagridKey` in addition to the 
+`resourceKey`. This allows to have the same endpoint for multiple datagrids, and solves a bunch of issues we were having.
```

Comment:
I would omit the part after the comma, it actually didn't solve any issues apart from the possibility to have multiple datagrids, as you have written right before.